src/System.Private.CoreLib/src/System/Globalization/FormatProvider.cs

```diff
@@ -95,11 +93,11 @@ public static String FormatDecimal(Decimal value, String format, IFormatProvider
         #endregion
 
         #region Parsing
-        public static Decimal ParseDecimal(String value, NumberStyles options, IFormatProvider provider)
+        public static Decimal ParseDecimal(ReadOnlySpan<char> value, NumberStyles options, IFormatProvider provider)
```

Comment:
This class is left over from unfinished refactoring. We are gradually getting rid of by changing callers to call the methods directly - to make the sources match CoreCLR.